src/entities/purchase-params.ts

```diff
@@ -21,4 +22,26 @@ export interface PurchaseParams {
    * The email of the user. If undefined, RevenueCat will ask the customer for their email.
    */
   customerEmail?: string;
+
+  /**
+   * Optional customization for the translations used in the purchase flow.
+   * The keys are the locale codes and the values are objects with the same keys as the default translations.
+   */
+  customTranslations?: CustomTranslations;
+
+  /**
+   * The locale to use for the purchase flow. If not specified, the device's locale will be used.
+   */
+  selectedLocale?: string;
+
+  /**
+   * Specifies if the locale should be picked automatically using the navigator.language property.
+   * Defaults to false.
+   */
+  useBrowserLocale?: boolean;
+
+  /**
+   * The default locale to use if the device's locale is not available.
+   */
+  defaultLocale?: string;
```

Comment:
The problem with that is that we will receive the language to use from the paywall in the renderPaywall method.

So basically no matter what's the SDK config we need to follow which language the paywall data passes to us when it will invoke the `purchase` method internally.

Reply:
Here's the point in which the paywall "passes" the locale and default locale to the purchase:
https://github.com/RevenueCat/purchases-js/pull/241/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR308-R309

Reply:
Hmm we could also move the locale for the `renderPaywall` method off that method and into configuration time I think?

Reply:
Not really because the data we receive for the supported and default locale are in the offering data itself.

Also why forcing this to be at configuration time?
Why do we need/want to add this there?